.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: trivy
+
+on:
+  workflow_call:
+    inputs:
+      image-ref:
+        default: "ghcr.io/${{ github.repository }}:${{ github.sha }}"
+        required: false
+        type: string
+
+jobs:
+  trivy:
+    name: Trivy Vulnerability Scan for images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
```

Comment:
The workflow is failing in the test, I believe that it's due to the lack of `docker login`. 
Checkout how the `docker-build.yaml` workflow handles the workflow login, I suggest you to do the same.

Reply:
I'm on it